ui/shared/tabs/failureSummary/SuggestionsListItem.jsx

```diff
@@ -110,7 +117,15 @@ export default class SuggestionsListItem extends React.Component {
           >
             <FontAwesomeIcon icon={faBug} title="File bug" />
           </Button>
-          <span>{suggestion.search}</span>
+          <a
```

Comment:
let's have these links include `target="_blank"` to create a new tab.  You'll also need `rel="noopener noreferrer"`

Reply:
Done

Reply:
Thanks!!